manim/mobject/three_dimensions.py

```diff
@@ -33,9 +37,22 @@ def __init__(self, shade_in_3d=True, **kwargs):
         super().__init__(shade_in_3d=shade_in_3d, **kwargs)
 
 
-class ParametricSurface(VGroup):
+class ParametricSurface(VGroup, metaclass=ConvertToOpenGL):
```

Comment:
This shouldn't be necessary since `VGroup`, which this class inherits from, already has `metaclass=ConvertToOpenGL` 
```suggestion
class ParametricSurface(VGroup):
```

Reply:
oh i thought daryl asked me to make this change; i must've misunderstood